### PR TITLE
HOTFIX: compile error in EmbeddedKafkaCluster

### PR DESCRIPTION
**Pull request by chia7712**
the signature of EmbeddedKafkaCluster#getAllTopicsInCluster was changed by #8062 but we did not change the usage introduced by #8029 before merging #8062

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by chia7712** [comment]
ping @junrao @vvcephei

**Review by omkreddy — APPROVED** [review]
@chia7712 Thanks for the PR. LGTM.

**Comment by omkreddy** [comment]
Merging to unblock the PR.

**Comment by vvcephei** [comment]
Ah, sorry about that, all. I should have checked the build again after merging. Thanks for the fix.

**Comment by highluck** [comment]
@omkreddy 
I didn't know that the registerWatch flag was added.
sorry, Thanks for the fix

**Pull request merged**
